guides/common/modules/proc_removing-ansible-roles-from-a-host.adoc

```diff
@@ -4,7 +4,9 @@
 Use the following procedure to remove Ansible roles from a host.
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and select the host you want to remove a role from.
-. On the *Ansible* tab, click the *Edit Ansible Roles* button.
-. In the *Edit Ansible Roles* window select the role you want to remove and click the *<* button to remove it.
-. Click *Confirm*.
+. In the {ProjectWebUI}, navigate to *Hosts > All Hosts*.
+. Select the host and click *Edit*.
+. On the *Ansible Roles* tab, navigate to the *Available Ansible Roles* area.
```

Comment:
```suggestion
. On the *Ansible Roles* tab, navigate to the *Assigned Ansible Roles* area.
```

Maybe also reword "navigate to" because this does not require a click.